packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -199,8 +199,8 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Dialog--fullscreen {
-  width: calc(100vw - 32px);
-  height: calc(100vh - 32px);
+  width: calc(100vw - 64px);
```

Comment:
where does 64 come from? should it change with scale?

Reply:
32px is the padding that is supposed to be around the fullscreen dialog. Since that padding is 32px for both top/bottom and left/right, the above needed to get doubled so it actually fit the 32px padding container

Reply:
Not sure if it should change with scale, didn't see fullscreen in the XD designs, I'll ask around

Reply:
Updated to 40px constant as per design: https://excelerate-team.slack.com/archives/G8R3V30FL/p1582920957021300